website/identifiers/clients/crossref.py

```diff
@@ -138,6 +138,23 @@ def build_posted_content(self, preprint, element, include_relation):
         ]
         posted_content.append(element.doi_data(*doi_data))
 
+        preprint_versions = preprint.get_preprint_versions()
+        for preprint_version in preprint_versions:
+            preprint_identifier = preprint_version.identifiers.first().value
+            if doi != preprint_identifier:
+                doi_relations = element.doi_relations(
+                    element.doi(doi),
+                    element.program(
+                        element.related_item(
+                            element.description('Updated version'),
+                            element.intra_work_relation(
+                                preprint_identifier,
+                                **{'relationship-type': 'isVersionOf', 'identifier-type': 'doi'}
+                            )
+                        ), xmlns=CROSSREF_RELATIONS
+                    )
+                )
+                posted_content.append(doi_relations)
         return posted_content
```

Comment:
Can you add a local output of this posted_content for the case where we have v1 and v2 and we are creating a v3? Thanks!